sgf-viewer/src/content.ts

```diff
@@ -284,6 +284,68 @@ export const pages: object =
                 "adversary": "Tony Wang (Author)",
                 "victim": "NeuralZ06 (KataGo KGS bot playing without search)",
                 "description": ["The same Go novice (Tony Wang) was able to exploit the top-50 KGS bot <code>NeuralZ06</code> by mimicking the behavior of our adversarial policy. The bot plays with checkpoint <code>b40c256-s11101799168-d2715431527</code> that is comparable to (but slightly weaker) than the <code>Latest</code> checkpoint. However, the bot has the <code>friendlyPassOk</code> flag enabled, which makes it easier to exploit—we have not been able to win manually against a bot with this disabled. This suggests that the easily mimicable high-level strategy of our adversarial policy explains a considerable part of the adversary's success, but by no means all of it.  We score the game under <a target=\"_blank\" href=\"https://tromp.github.io/go.html\">Tromp-Taylor</a> rules as the rulesets supported by KGS cannot be automatically evaluated."]
+            },
+            {
+                "title": "Human exploits KataGo",
+                "dir_name": "human_vs_kata100k",
+                "server": "dqn.ist.berkeley.edu",
+                "paths": [
+                    "/nas/ucb/kpelrine/humanatk/humanattack-KataGo-100kvisits.sgfs"
+                ],
+                "max_games": 10,
+                "adversary": "Kellin Pelrine (Author)",
+                "victim": "KataGo, 100K visits",
```

Comment:
should we specify that this is with `<code>Latest</code>` as the network? Or was this on KGS and we don't know which network it was?

Reply:
I checked and I used b18c384nbt-uec.bin.gz from https://github.com/lightvector/KataGo/releases/tag/v1.12.0. From the description this is comparable to 60 block while our Latest is 40 block, and it's also newer, so this should actually be stronger than our Latest. Adds a bit more diversity too. I think maybe not ideal to have a bunch of numbers that won't be understandable in the "victim" field, but I'll add a more informative note to the text.